logql/v2/ast.go

```diff
@@ -61,17 +61,15 @@ func (s *StreamMatcherExpr) Walk(fn WalkFn) {
 func (s *StreamMatcherExpr) String() string {
 	var sb strings.Builder
 
-	sb.WriteString("{")
-
+	sb.WriteRune('{')
```

Comment:
What is the value of this? I am just curious.

Reply:
I assume it's supposed to be more optimized but I admin I have no certitude I could not find a clear statement on that while googling, and doing a quick benchmark isn't conclusive. I can revert then ...

Reply:
Let's keep it the old way